src/System.Private.CoreLib/shared/System/Globalization/CultureData.cs

```diff
@@ -794,9 +796,9 @@ internal string CultureName
                 {
                     case "zh-CHS":
                     case "zh-CHT":
-                        return _sName;
+                        return _sName!;
```

Comment:
This seems like a compiler analysis bug.  We just did a switch on _sName and successfully matched it against a non-null string, so it can't be null.
cc: @jaredpar

Reply:
This is a bug. This PR is fixing up the logic around `switch` expressions and statements to reflect the LDM decisions from the previous few weeks. [https://github.com/dotnet/roslyn/pull/34273](https://github.com/dotnet/roslyn/pull/34273)

---
In reply to: [270458508](https://github.com/dotnet/coreclr/pull/23526#discussion_r270458508) [](ancestors = 270458508)